src/TrezorKeyring.test.ts

```diff
@@ -131,22 +140,31 @@ describe('TrezorKeyring', function () {
     it('constructs', function (done) {
       const t = new TrezorKeyring({ hdPath: `m/44'/60'/0'/0` });
       assert.equal(typeof t, 'object');
-      t.getAccounts().then((accounts) => {
-        assert.equal(Array.isArray(accounts), true);
-        done();
-      });
+      t.getAccounts()
+        .then((accounts) => {
+          assert.equal(Array.isArray(accounts), true);
+          done();
+        })
+        .catch((e) => {
+          throw new Error(e?.toString() ?? 'Unknown error');
+        });
```

Comment:
Hmm, why introduce this catch function if we're just going to re-throw? We could leave this out and let the error bubble up directly.

We're losing information from the error here too (e.g. the stack trace); only the error message is passed with this new error.

Reply:
I added those initially because I had some mess with eslint that was throwing an error for dangling promises, but I forgot to remove them after fixing it! 

I removed all of them.